app/components/task_list_items/consultee_responses_component.rb

```diff
@@ -19,7 +19,9 @@ def link_text
     end
 
     def link_active?
-      true
+      return true unless planning_application.pre_application?
+
+      planning_application.consultation_required?
```

Comment:
I find these guard clauses a little hard to follow. Would it perhaps be clearer as something like:

```suggestion
      if planning_application.pre_application?
        planning_application.consultation_required?
      else
        true
      end
```

This should be the same in the other components.

But also, the fact that it's repeated in three components makes me think it might be in the wrong place.

Perhaps the `consultation_required?` method should return true for all applications except pre-apps, and true/false for pre-apps as appropriate, and then the components only need to call that method? i.e.:

```suggestion
      planning_application.consultation_required?
```